src/oc/web/actions/jwt.cljs

```diff
@@ -15,7 +16,7 @@
 
 (defn logout
   ([]
-   (logout (if js/window.OCCarrotDesktop
+   (logout (if ua/pseudo-native?
```

Comment:
@cjsauer you sure this redirect should be changed for both desktop and mobile wrappers? Before it was only for desktop if i understand the removed code...

Reply:
@bago2k4 yes it's correct. Both desktop and mobile need to redirect to the "desktop login" page, which is really just a plain login page. The name `desktop-login` is kind of unfortunate. I think we should rename it here.